app/src/AddItemForm.js

```diff
@@ -0,0 +1,103 @@
+import "./App.css";
+
+import React, { useState, useEffect, useRef } from "react";
+
+import * as apiClient from "./apiClient";
+
+const AddItem = ({ onAdd }) => {
+  const inputItem = useRef();
+  const [name, setName] = useState("");
+  console.log(name);
+
+  const addNewItem = async (name) => {
+    const response = await apiClient.addItem(name);
+    onAdd(response[0]);
+    // window.location = "/";
+    console.log(response);
+  };
+  // Clicking Edit/on the value activates editMode. User sees input bar and Submit button.
+  const onEditClick = () => {
+    setEditMode(true);
+  };
+
+  // Submit sets setEditMode(false). User sees value with submitted input item.
+  const onSubmitClick = () => {
+    addNewItem(name);
+    setName("");
+    setEditMode(false);
+  };
+
+  // Want: When user clicks out => setEditMode(false) => editMode === false, a line with original value.
```

Comment:
What do you mean by "when user clicks out" ?

Reply:
Like when user clicks outside of the input box!